src/WebUI/WorkLoads/WorkloadDetails.tsx

```diff
@@ -0,0 +1,207 @@
+                renderCell: WorkloadDetails._renderImageCell
+            },
+            {
+                id: "w-labels",
+                name: Resources.LabelsText,
+                width: -100,
+                className: "w-key",
+                minWidth: 200,
+                headerClassName: "workload-details-column-header",
+                renderCell: WorkloadDetails._renderLabelsCell
+            }
+        ];
+        return columns;
+    }
+
+    private _getWorkloadDetails(): JSX.Element | null {
+        const metadata = this.props.parentMetaData;
+        if (metadata) {
+            const pipeline = Utils.getPipelineText(metadata.annotations);
+            const tableItems = new ArrayItemProvider<any>([{}, { podTemplate: this.props.podTemplate, parentMetaData: metadata }]);
+            const agoTime = Date_Utils.ago(new Date(metadata.creationTimestamp), Date_Utils.AgoFormat.Compact);
+            return (
+                <div className="kube-list-content s-details depth-16">
+                    <div className="workload-details-primary-text">{localeFormat(Resources.WorkloadDetails, this.props.parentKind)}</div>
+                    <div className="workload-details-secondary-text">
+                        {pipeline ? localeFormat(Resources.ServiceCreatedText, agoTime, pipeline) :
+                            localeFormat(Resources.CreatedAgo, agoTime)}
+                    </div>
+                    <Table
+                        className="w-details"
+                        id={format("workload-details-tbl-{0}", metadata.uid)}
+                        showLines={false}
+                        showHeader={false}
+                        singleClickActivation={false}
+                        itemProvider={tableItems}
+                        pageSize={tableItems.length}
+                        columns={WorkloadDetails._getColumns()}
+                    />
+                </div>
+            );
+        }
+
+        return null;
+    }
+
+    private _getAssociatedPods(): JSX.Element | null {
+        if (this.state.pods.length === 0) {
+            return KubeZeroData._getDefaultZeroData("https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/",
+                Resources.LearnMoreText, Resources.NoPodsFoundText,
+                Resources.LinkSvcToPodsText, Resources.PodsText);
+        }
+
+        return (
+            <PodsTable
+                podsToRender={this.state.pods}
+                headingText={Resources.PodsText}
+                onItemActivated={this._onSelectedPodInvoked}
+            />
+        );
+    }
+
+    private _onSelectedPodInvoked = (event: React.SyntheticEvent<HTMLElement>, pod: V1Pod) => {
+        this.setState({
+            showSelectedPod: true,
+            selectedPod: pod
+        });
+    }
+
+    private static _renderImageCell(rowIndex: number, columnIndex: number, tableColumn: ITableColumn<any>, tableItem: any): JSX.Element {
```

Comment:
as per spec this is link

Reply:
added to do to make it a link when we have image details view